docker-compose.yml

```diff
@@ -27,14 +27,14 @@ services:
     depends_on:
       - "mqtt_server"
 
-  windwow:
-    ports:
-        - '9084:9080'
-    restart: always
-    image: windwow:latest
-    build: ./WindWow
-    depends_on:
-      - "mqtt_server"
+  #windwow:
```

Comment:
I wasn't able to clone this project and I can't find https://github.com/unibuc-cs/WindWow on github.
Was it deleted?

Reply:
It's private. Talk to @raresito to change its permissions.